doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_pm_optimization.rst

```diff
@@ -64,6 +64,127 @@ The power management subsystem in a local domain is responsible for scheduling a
+
+* Use auxiliary or peripheral-oriented cores (for example, PPR- or FLPR-type helper cores) to perform SPI, I2C, ADC, or UART transactions, local preprocessing, and DMA into local RAM.
+* Signal (IPC or shared memory flag) the application or radio core only when a batch of data is ready.
+* Avoid fine-grained peripheral polling from high-performance cores.
+  Use the following instead:
+
+  * DMA transfers initiated by auxiliary cores.
+  * Hardware-trigger chains (PPI/DPPI) where possible to reduce wake-ups.
+
+* Keep high-speed power domains off unless necessary for bursts of compute.
+
+DMA locality constraints
+------------------------
+
+Local DMA controllers (for example, per-domain ECDMA instances) are restricted to source/destination addresses within their local RAM range.
+Plan buffer placement accordingly:
+
+* Peripheral DMA to/from local domain RAM only.
+* Cross-domain transfers should use the following:
+
+  * IPC messages indicating buffer readiness.
+  * Explicit software copy (only for infrequent cases).
+* Avoid placing frequent DMA targets in global RAM (MRAM).
+  Use retained local RAM to keep transfers local and reduce inter-domain wake-ups and bus activity.
+
+System-off entry prerequisites and wake-up sources
+--------------------------------------------------
+
+To reach the lowest power (system-off) state, all non-essential wake-up capable peripherals must be disabled.
+Only a limited set of wake-up sources should remain enabled.
+Retention of selected RAM blocks is optional and increases leakage.
+Retain only what is required for fast resume.
+
+Typical system-off wake-up sources are the following:
+
+* Port **0** GPIO (configured for wake-up, always powered pad group)
+* GRTC compare/channel event
+* LPCOMP (low-power comparator)
+* NFC field detection
+* USB power presence (VBUS connect/disconnect)
+* (Optional) Specific voltage detect events
+
+To optimize system-off entry, do the following:
+
+* Before requesting system-off, disable or runtime-suspend all other devices to allow hardware to drop unused power domains.
+* Audit enabled interrupts.
+  Any enabled interrupt tied to an active peripheral can prevent deep low-power entry.
+* Use runtime PM (using :kconfig:option:`CONFIG_PM_DEVICE_RUNTIME` and the ``zephyr,pm-device-runtime-auto`` DTS node) to ensure peripherals are suspended before system-off.
+* Ensure that only the required GPIOs remain configured with sense settings.
+  Remove sense from unused pins.
+
+The following is an example sequence:
+
+1. Flush any pending work and commit the application context to the retained low-leakage RAM.
+#. Suspend peripherals (runtime PM).
+#. Configure minimal wake-up sources (for example, one GRTC channel and a GPIO).
+#. Request system-off (``pm_state_force`` or appropriate API).
+#. On wake, the bootloader resumes and the application restores context.
+   If MCUboot Suspend-to-RAM (S2RAM) is enabled, the MCUboot S2RAM path handles the resume process.
```

Comment:
System-off doesn't restore context. After wake-up it just starts app again (just like standard reset). 
When it comes to S2RAM it is triggered by `k_sleep(n)` , if `n` is  higher, than [min-residency-us](https://github.com/nrfconnect/sdk-zephyr/blob/main/dts/vendor/nordic/nrf54h20.dtsi#L149) .
Also if MCUboot is enabled, the procedure is as follows:
 - if MCUboot detects returning from S2RAM, it just sets the PC to application's reset handler (it doesn't handle the resume process)
 - if the application's reset handler detects returning from S2RAM it handles the resume process
 - After that application continues execution.